tests/test_notify_buyers_when_requirements_close.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+import mock
+from freezegun import freeze_time
+
+import datetime
+
+from dmscripts.notify_buyers_when_requirements_close import (
+    get_closed_briefs,
+    get_closed_at,
+    get_notified_briefs,
+    notify_users,
+    MandrillException,
+    main
+)
+
+from .helpers import FakeMail
+
+
+def test_get_closed_briefs_filters_by_closed_at():
+    api_client = mock.Mock()
+    api_client.find_briefs_iter.return_value = iter([
+        {"applicationsClosedAt": "2016-09-03T23:59:59.000000Z", "status": "closed"},
+        {"applicationsClosedAt": "2016-09-04T23:59:59.000000Z", "status": "closed"},
+        {"applicationsClosedAt": "2016-09-05T23:59:59.000000Z", "status": "closed"},
+        {"applicationsClosedAt": "2016-09-06T23:59:59.000000Z", "status": "closed"},
+        {"applicationsClosedAt": "2016-09-07T23:59:59.000000Z", "status": "closed"},
+        {"applicationsClosedAt": "2016-09-08T23:59:59.000000Z", "status": "closed"},
+        {"applicationsClosedAt": "2016-09-05T23:59:59.000000Z", "status": "closed"},
+        {"applicationsClosedAt": "2016-09-09T23:59:59.000000Z", "status": "closed"},
+        {"applicationsClosedAt": "2016-09-05T23:59:59.000000Z", "status": "closed"},
+    ])
+
+    assert get_closed_briefs(api_client, datetime.date(2016, 9, 5)) == [
+        {"applicationsClosedAt": "2016-09-05T23:59:59.000000Z", "status": "closed"},
```

Comment:
Might be nice to mix up the time of day a bit?  Not that it makes any real difference.